chat-relay/package.json

```diff
@@ -7,8 +7,11 @@
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
     "dotenv": "^10.0.0",
```

Comment:
I would say to remove these as I don't think you need to have these unless you're setting up a different HTTP server for proxying.